lang/LangPrimSource/PyrPrimitive.cpp

```diff
@@ -1026,6 +1026,10 @@ int objectPerformArgsAndKwArgs(struct VMGlobals* g, int numArgsPushed) {
 
     const auto argsSize = argsArray ? argsArray->size : 0;
     const auto kwSize = kwargsArray ? kwargsArray->size : 0;
+    if (kwSize % 2 == 1) {
+        error("Should be an even number of keyword arguments as they are Symbol value pairs.\n");
```

Comment:
Better: "Keyword or value is missing. An even number of keyword arguments should be passed."

Or, like below:
"WARNING: Keyword arguments must be a key-value pair array, where the key is a Symbol.\n"

?